src/base/utils/compare.h

```diff
@@ -29,6 +29,7 @@
 #pragma once
 
 #include <Qt>
+#include <QtCore/private/qtcore-config_p.h>  // for QT_FEATURE_xxx
```

Comment:
Doesn't seem like it is intended to be used in applications. Isn't there any better way to do the job instead of this workaround?

Reply:
>Isn't there any better way to do the job instead of this workaround?

I started from this page and the result is this PR: https://wiki.qt.io/Qt5_Build_System#How_to

Reply:
> I started from this page and the result is this PR: https://wiki.qt.io/Qt5_Build_System#How_to

I see. And it looks confusing to me.
Well, let's do it this way for now.

Reply:
I would put this private header include separate from regular ones. And maybe add the link provided above to the comment near it.

Reply:
>I would put this private header include separate from regular ones. And maybe add the link provided above to the comment near it.
Done.